### ISSUE TITLE
rexify --bundle recursively copies its payload on Darwin

### ISSUE THREAD

**Issue by mbroadhead** [RexOps/Rex#826]
The `rex --bundle` command executes `cp -vR * rex.payload/`. With GNU `cp`, it prevents copying directories into itself (tested on CentOS 6):
```
cp: cannot copy a directory, `rex.payload', into itself, `rex.payload/rex.payload'
```

With Darwin's `cp` command, it does not prevent this, so you get stuck in an infinite copy loop.

We could use `rsync -av --exclude=rex.payload * rex.payload/` instead of using `cp` to fix this. Is there a better way?

Here is my setup:
```
$ uname -v
Darwin Kernel Version 15.0.0: Sat Sep 19 15:53:46 PDT 2015; root:xnu-3247.10.11~1/RELEASE_X86_64

$ rex -v
(R)?ex 1.3.3
```

**Comment by mbroadhead** [comment]
It looks as though `rex --bundle` probably won't work on Darwin anyway, since it is unlikely that it will find `libexpat.so` and `libssh2.so`. We can probably just backlog this ticket for later until there is better Darwin support.

**Comment by krimdomu** [comment]
@mbroadhead yes, currently the bundle is only supported on 64bit debian. I don't have a Darwin system where i can develop this feature.
